packages/ui/src/components/molecules/copy-to-clipboard/copy-to-clipboard.tsx

```diff
@@ -0,0 +1,78 @@
+"use client";
+
+import React, { useCallback, useMemo, useState } from "react";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "../../atoms";
+
+interface CopyToClipboardProps extends React.PropsWithChildren {
+  value: string;
+  content?: React.ReactNode;
+  side?: "left" | "right" | "top" | "bottom" | undefined;
```

Comment:
I think the question mark after the property name makes the `| undefined` redundant.